lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "hoist": true,
+  "hoist": false,
```

Comment:
you're working with true, right?

Reply:
yep

Reply:
so it's not necessary here

Reply:
its kind are bug, this change was mark outdated

Reply:
ohhh.... Sorry